cloudwatch_logs.tf

```diff
@@ -18,6 +18,7 @@ data "aws_iam_policy_document" "cloudwatch_logs_policy" {
 }
 
 resource "aws_iam_policy" "cloudwatch_logs_policy" {
+  name   = "cw-logs-access-${var.service_name}"
```

Comment:
Policy names need to be unique, so I would suggest that we include the region inside the name to support multi-region usage of a fargate service. Something like `"${var.service_name}-cw-logs-${data.aws_region.current.name}"`